html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,53 @@
 <?php
-$limit = $_GET['target'];
+$limit = mb_convert_kana($_GET['target'], 'a', 'UTF-8');
+if(!is_numeric($limit) || $limit <= 0 || preg_match('/^([1-9]\d*|0)\.(\d+)?$/', $limit)) {
+  http_response_code(400);
+  echo ('1以上の整数をターゲット値としてください');
```

Comment:
要件には無いですが、せっかくなのでレスポンスはjson形式に統一しましょう

```suggestion
  echo json_encode('1以上の整数をターゲット値としてください', JSON_UNESCAPED_UNICODE);
```

Reply:
ご提案ありがとうございます。実装いたしました。